sys/netpfil/ipfilter/netinet/ip_nat.c

```diff
@@ -7352,30 +7347,18 @@ ipf_nat_nextaddr(fr_info_t *fin, nat_addr_t *na, u_32_t *old, u_32_t *dst)
 {
 	ipf_main_softc_t *softc = fin->fin_main_soft;
 	ipf_nat_softc_t *softn = softc->ipf_nat_soft;
-	u_32_t amin, amax, new;
+	u_32_t new;
 	i6addr_t newip;
 	int error;
 
 	new = 0;
-	amin = na->na_addr[0].in4.s_addr;
 
 	switch (na->na_atype)
 	{
 	case FRI_RANGE :
-		amax = na->na_addr[1].in4.s_addr;
-		break;
-
 	case FRI_NETMASKED :
 	case FRI_DYNAMIC :
 	case FRI_NORMAL :
-		/*
-		 * Compute the maximum address by adding the inverse of the
-		 * netmask to the minimum address.
-		 */
-		amax = ~na->na_addr[1].in4.s_addr;
-		amax |= amin;
-		break;
-
```

Comment:
This is indeed unused. I don't know if this indicates a problem or not. Why was this computed before? Can we trace back to where its use was removed? The tangled history of this port / software isn't hepful. this appeared from an upstream import of
`Import IP-Filter 5.1.2 into vendor branches using the existing license that`
from 2013. It wasn't in any prior version, so I think it's likely safe. It won't break anything that's not already broken and this code will still be in git if someone discovers that it should have been used and needs to recover it.

Reply:
Agreed.